### PR TITLE
Add getTypeId

### PR DESCRIPTION
**Pull request by chenyang14**

### CODE REVIEW

File: arrange_schedule.py [hunk 1]
```diff
@@ -861,10 +861,10 @@ def read_arrange_mode():
 
 def find_cwb_type_id(db):
     return_msg = {}
-    sql = "SELECT type_id FROM data_type WHERE type_name='氣像雲圖'"
-    pure_result = db.query(sql)
-    if len(pure_result) != 0:
-        return int(pure_result[0][0])
+    with DataTypeDao() as dataTypeDao:
+        typeid = dataTypeDao.getTypeId('氣像雲圖')
```
Comment (Billy4195) [inline arrange_schedule.py]: The variable's name should use ***lower camel case*** or split the nouns by `_`

File: arrange_schedule.py [hunk 2]
```diff
@@ -1072,10 +1072,10 @@ def crawler_google_drive_img(json_obj):
         db.connect()
 
         #find google_drive_image type id 
-        sql = "SELECT type_id FROM data_type WHERE type_name='google_drive_image'"
-        pure_result = db.query(sql)
+        with DataTypeDao() as dataTypeDao:
+            typeid = dataTypeDao.getTypeId('google_drive_image')
```
Comment (Billy4195) [inline arrange_schedule.py]: The problem is same as above

File: arrange_schedule.py [hunk 3]
```diff
@@ -861,10 +861,10 @@ def read_arrange_mode():
...
 def find_cwb_type_id(db):
     return_msg = {}
-    sql = "SELECT type_id FROM data_type WHERE type_name='氣像雲圖'"
-    pure_result = db.query(sql)
-    if len(pure_result) != 0:
-        return int(pure_result[0][0])
+    with DataTypeDao() as dataTypeDao:
+        typeid = dataTypeDao.getTypeId('氣像雲圖')
+    if typeid != None:
+        return int(typeid)
```
Comment (Billy4195) [inline arrange_schedule.py]: I think the `typeid` should cast to int() in the DataTypeDao method

File: arrange_schedule.py [hunk 4]
```diff
@@ -1072,10 +1072,10 @@ def crawler_google_drive_img(json_obj):
         db.connect()
 
         #find google_drive_image type id 
-        sql = "SELECT type_id FROM data_type WHERE type_name='google_drive_image'"
-        pure_result = db.query(sql)
+        with DataTypeDao() as dataTypeDao:
+            typeid = dataTypeDao.getTypeId('google_drive_image')
         try:
-            data_type = int(pure_result[0][0])
+            data_type = int(typeid)
```
Comment (Billy4195) [inline arrange_schedule.py]: Same as line 867

**Comment by chenyang14** [comment]
Variables change to lower camel case
Since typeId is int(), omit the step to cast to int().

**Pull request merged**